libs/hbb_common/protos/message.proto

```diff
@@ -124,11 +124,17 @@ message PeerInfo {
   string platform_additions = 12;
 }
 
+message RdpUserSession {  
+  string user_session_id = 1;  
+  string user_name = 2;  
+}  
+
 message LoginResponse {
   oneof union {
     string error = 1;
     PeerInfo peer_info = 2;
   }
+  repeated RdpUserSession rdp_user_sessions = 3;  
```

Comment:
`rdp_user_sessions` can be moved out of `LoginResponse`

Reply:
Where should I place it? I tried placing it in `Message` it gives error `label not allowed`.
I was asked to use repeated here https://github.com/rustdesk/rustdesk/pull/6825#discussion_r1445021973
Also, if I move it out will this part work? As this is part of login response.
```rust
      res.rdp_user_sessions = rdp;
      let mut msg_out = Message::new();
      msg_out.set_login_response(res);
      self.send(msg_out).await;

Reply:
The login has ended. This can be treated as a normal message. You can try putting it in Misc.